data/CrNiP/UppASD/MC/inpsd.dat

```diff
@@ -0,0 +1,60 @@
+simid bccFe100                                    
+ncell     25       25      25                System size            
+BC        P         P         P                 Boundary conditions (0=vacuum, P=periodic)
+cell          5.0575886562   -2.9200001721    0.0000000000
+              0.0000000000    5.8400003443    0.0000000000
+              0.0000000000    0.0000000000    3.5140002072
+Sym       0                                     Symmetry of lattice (0 for no, 1 for cubic, 2 for 2d cubic, 3 for hexagonal)
+
+posfile   ./posfile
+posfiletype D
+momfile   ./momfile
+exchange  ./jASD2
+#anisotropy ./kfile
+
+do_ralloy 0
+Mensemble 1
+tseed 4499
+maptype 3
```

Comment:
@avishina What is the meaning of `maptype 3`? The [UppASD documentation](https://uppasd.github.io/UppASD-manual/input/) only explains types 1 and 2:
```quote
For 1=coordinates the coordinates are given in Cartesian or direct coordinates (see posfiletype).

For 2 the coordinates of a coupling vector are implicitly given by specifying that the coupling links atom with atom (for an example, see dm).
```